libraries/cli/include/cli/config_jsons/devnet/genesis.json

```diff
@@ -0,0 +1,276 @@
+{
+  "dag_genesis_block": {
+    "level": "0x0",
+    "pivot": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "sig": "0xb7e22d46c1ba94d5e8347b01d137b5c428fcbbeaf0a77fb024cbbf1517656ff00d04f7f25be608c321b0d7483c402c294ff46c49b265305d046a52236c0a363701",
+    "timestamp": "0x60aee104",
+    "tips": [],
+    "transactions": []
+  },
+  "state": {
```

Comment:
not sure here, but didnt you want to remove whole state object ? I think we could possibly get rid of that too ?

Reply:
So I changed jsons and way we are processing it. But we still have it in the code, because it is easier to work with it like this